backend/Gemfile

```diff
@@ -12,6 +12,9 @@ gem "rake"
 # JSON serializer
 gem "active_model_serializers", "0.9.8"
 
+# Brakeman security scanner
+gem "brakeman", "6.1.2"
```

Comment:
Should this be a development dependency only? I don't think we want this installed in production.

Reply:
Good point!